test/WebJobs.Script.Tests.Integration/Host/StandbyManagerTests.cs

```diff
@@ -112,21 +112,29 @@ public void IsWarmUpRequest_ReturnsExpectedValue()
 
                 // Now specialize the host
                 ScriptSettingsManager.Instance.SetSetting(EnvironmentSettingNames.AzureWebsitePlaceholderMode, "0");
-                request = new HttpRequestMessage(HttpMethod.Get, "api/dne");
```

Comment:
We were relying on the http activation to force the host to start after specialization, as you can see in the previous test code. Now we're relying totally on the specialization timer, and I verify in logs below that the host has been started after specialization.